docs/development/doc_howto.rst

```diff
@@ -110,17 +110,6 @@ Generating the HTML docs for Gammapy is straight-forward::
     make docs-all
     make docs-show
 
-Generating the PDF docs is more complex.
```

Comment:
Making the pdf no longer works?

Reply:
I have modified the text of the recipe on how to build the docs as a PDF file.
The old recipe did not work since a long time...